config/settings.yaml.example

```diff
@@ -17,6 +17,10 @@
 # Mark translated strings with X characters (for developers)
 :mark_translated: false
 
+# Disable the webpack development server (for developers)
+# Make sure to run `rake webpack:compile` if disabled.
+#:webpack_dev_server: false
```

Comment:
Nitpick, this should show the default/current value rather than being commented out as `false`, it'll be clearer. The comment above may need changing too.